frontend/src/scenes/project/Settings/AccessControl.tsx

```diff
@@ -6,25 +6,35 @@ import { useActions, useValues } from 'kea'
 import { RestrictedComponentProps } from '../../../lib/components/RestrictedArea'
 import { sceneLogic } from '../../sceneLogic'
 import { teamLogic } from '../../teamLogic'
+import { LockOutlined, UnlockOutlined } from '@ant-design/icons'
 
 export function AccessControl({ isRestricted }: RestrictedComponentProps): JSX.Element {
     const { currentOrganization, currentOrganizationLoading } = useValues(organizationLogic)
     const { currentTeam, currentTeamLoading } = useValues(teamLogic)
     const { updateCurrentTeam } = useActions(teamLogic)
     const { guardAvailableFeature } = useActions(sceneLogic)
 
+    const projectPermissioningEnabled =
+        currentOrganization?.available_features.includes(AvailableFeature.PROJECT_BASED_PERMISSIONING) &&
+        currentTeam?.project_based_permissioning
+
     return (
         <div>
-            <h2 className="subtitle" id="access-control">
-                Access Control
+            <h2 className="subtitle" id="privacy">
+                Privacy settings
```

Comment:
I'd prefer to stick to "Access Control". "Privacy" can be confused for user privacy (GDPR and such), while "Access Control" clearly refers to project access.